JankStatsSample/build.gradle

```diff
@@ -14,7 +24,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.4"
+        classpath 'com.android.tools.build:gradle:7.1.1'
```

Comment:
Latest stable is 7.1.3 (because I was so slow)

Reply:
Updated to 7.2.0 👍🏼